pkgs/tools/misc/copier/default.nix

```
@@ -0,0 +1,61 @@
+{ lib, gnused, poetry, git, python3, fetchFromGitHub }:
```

```suggestion
{ lib, poetry, git, python3, fetchFromGitHub }:
```
sed is part of the stdenv.